Gopkg.toml

```diff
@@ -57,7 +57,8 @@ required = [
 
 [[constraint]]
   name = "github.com/lyft/flyteplugins"
-  version = "^0.2.5"
+  # version = "^0.2.5"
+  branch = "simpler-hive-plugin-init"
```

Comment:
use a release version?

Reply:
https://github.com/lyft/flyteplugins/pull/44

Reply:
I'll update it after that PR is merged